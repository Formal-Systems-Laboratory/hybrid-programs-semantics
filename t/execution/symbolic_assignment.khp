vars a ; a := a + 1.0 ; a := *
