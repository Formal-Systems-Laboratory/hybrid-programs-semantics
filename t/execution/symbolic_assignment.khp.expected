<generatedTop>
  <k>
    .
  </k>
  <pgmVars>
    a , .Ids
  </pgmVars>
  <state>
    a |-> #VarReal ( a0 )
  </state>
  <evolutionConditions>
    .Set
  </evolutionConditions>
  <nonDetAssignments>
    a |-> #VarReal ( a0 )
  </nonDetAssignments>
  <counter>
    1
  </counter>
  <constraints>
    .
  </constraints>
</generatedTop>
